packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -0,0 +1,155 @@
+import { basename, dirname, extname, join } from "path";
+import { copy, readJson, writeJson } from "fs-extra";
+import { promises } from "fs";
+import { rm } from "shelljs";
+
+export function collectDependencies(
+    externals,
+    outputDir,
+    shouldCopyNodeModules = true,
```

Comment:
it doesn't make sense to have parameters with default value before parameters without it

Reply:
Oops, it was the last and then i moved.. fixed